pset-functions.js

```diff
@@ -148,6 +241,43 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+
+function doubleLetters(word) {
+  let letters = word.split('')
+
+  for (let i = 0; i < letters.length - 1; i++){
+    let previousLetter = letters[i]
+    let currentLetter = letters[i + 1]
+
+    let result = previousLetter===currentLetter
```

Comment:
Great use of a loop's iteration to check adjacent elements in a string. Way to keep in mind potential edge cases on line 246! For your benefit, please find an alternate solution [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L217-L234)).